grafana-plugin/src/containers/MobileAppConnection/MobileAppConnection.tsx

```diff
@@ -29,7 +32,28 @@ const INTERVAL_POLLING = 5000;
 const BACKEND = 'MOBILE_APP';
 
 const MobileAppConnection = observer(({ userPk }: Props) => {
-  const { userStore } = useStore();
+  const { userStore, cloudStore, backendLicense } = useStore();
+
+  // Show link to cloud page for OSS instances with no cloud connection
+  if (backendLicense === GRAFANA_LICENSE_OSS && !cloudStore.cloudConnectionStatus.cloud_connection_status) {
+    return (
+      <VerticalGroup spacing="lg">
+        <Text>Please connect Cloud OnCall to use the mobile app</Text>
+        {isUserActionAllowed(UserActions.OtherSettingsWrite) ? (
+          <PluginLink query={{ page: 'cloud' }}>
+            <Button variant="secondary" icon="external-link-alt">
+              Connect Cloud OnCall
+            </Button>
+          </PluginLink>
+        ) : (
+          <Text>
```

Comment:
This should probably take a `type="secondary"` as otherwise it adds `undefined` in the snapshot on the class. Same for the other `<Text />` above.